src/components/app/ServiceWorkerManager.js

```diff
@@ -66,75 +130,198 @@ class ServiceWorkerManager extends PureComponent<Props, State> {
     });
   }
 
-  componentDidMount() {
-    if (process.env.NODE_ENV === 'production') {
-      this._installServiceWorker();
+  _hasDatasourceProfile(): boolean {
```

Comment:
s-s-s-super nit: I think this should be capitalized like this to be consistent `_hasDataSourceProfile`